docs/detections/api/rules/rules-api-create-rule-default-exception-list.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[exceptions-api-create-rule-default-exception-list]]
+=== Create default exception list for a rule
+
+Creates a default exception list for the rule you specify. Default exception lists can only be associated with a single rule.
+
+To add exception items to a default exception list, pass in exceptions items that you want applied to the rule. Refer to <<exceptions-api-create-rule-default-exception-item>> for more information. 
+
+When an exception item’s query evaluates to `true`, the associated rule does not issue alerts even when its other criteria are met.
+
+NOTE: Default exception lists do not display on the <<shared-exception-lists, Shared Exception Lists>> page in the {security-app} UI. 
```

Comment:
Should we add where they do show up? Just in the rule details, exceptions tab?

Reply:
Added in [585ed2c](https://github.com/elastic/security-docs/pull/4021/commits/585ed2cb75a83d00203852718b0a7d5fb610f4f6).